Player.py

```diff
@@ -22,6 +22,10 @@ def __init__(self, x, y, playerId):
         self.y = y
         self.speed = 3
         self.state = 0
+        self.bombs = []
+        self.newBomb = 0
+        self.bombNumber = 10
+        self.bombPower = 3
```

Comment:
`bombNumber` is a maximum amount of bombs or amount of bombs that players currently holds? Change its name

Reply:
And what is `newBomb` ?

Reply:
The naming should be more clear now